README.md

```diff
@@ -1,2 +1,71 @@
-# terramate-plain-terraform-example
-Repository terramate-plain-terraform-example
+# initial state
+
+Four directories containing Terraform files describing a VPC and an Instance as Null Provider Resources for two environments `staging` and `prod`.
+
+- `environments/prod/vpc`
+- `environments/prod/ec2-instance`
+- `environments/staging/vpc`
+- `environments/staging/ec2-instance`
+
+# world with terramate
+
+- rename terramate config to disable production safeguards for this demo
+  - `cp terramate.tm.hcl.example terramate.tm.hcl`
```

Comment:
terramate.tm.hcl already exists, so either remove the terramate.tm.hcl.example file, or remove this step

Reply:
removed